coreapps-web-2.2/pom.xml

```diff
@@ -0,0 +1,77 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+      <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>coreapps</artifactId>
+        <version>1.32.0-SNAPSHOT</version>
+     </parent>
+     <artifactId>coreapps-web-2.2</artifactId>
+     <packaging>jar</packaging>
+     <name>coreapps Module web 2.2</name>
+     <description>2.2 web project for coreapps</description>
+     
+     <properties>
+        <openMRSVersion>2.5.0-SNAPSHOT</openMRSVersion>
+    
+     </properties>
+     
+     <dependencies>
+     
+      <!-- Begin OpenMRS core -->
+
+        <dependency>
```

Comment:
@ibacher  @mozzy11  thanks mosses dropping the service since its not needed and using @component has helped, I cleaned my dependencies as suggested by @ibacher  am now making sure my tests are correct then i finalize on all the required changes